ntfs_fr.md

```diff
@@ -528,9 +529,24 @@ Colonne | Type | Contrainte | Commentaire
 address_id | chaine | Requis | Identifiant de l'adresse
 street_name | chaine | Requis | Nom de la voierie
 house_number | chaine | Optionnel | Numéro du seuil
+admin_level_8_id | chaine | Optionnel | Identifiant de l'administration de niveau 8 dans le fichier `administrative_regions.txt`
+admin_level_9_id | chaine | Optionnel | Identifiant de l'administration de niveau 9 dans le fichier `administrative_regions.txt`
+admin_level_10_id | chaine | Optionnel | Identifiant de l'administration de niveau 10 dans le fichier `administrative_regions.txt`
+
+### administration_regions.txt (optionnel)
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+admin_id | chaine | Requis | Identifiant de la région administrative
+admin_insee | chaine | Requis | Code INSEE
+admin_level | entier | Requis | Niveau de la région administrative
+admin_name | chaine | Requis | Nom de la région administrative
+admin_label | chaine | Requis | Label de la région administrative
+admin_zip_codes | chaine | Requis | Liste de code postaux séparés par un point virgule
```

Comment:
not sure

Reply:
```suggestion
admin_zip_codes | chaine | Requis | Liste de code postaux 
```
so you don't make a promise you are not sure to keep :)

Reply:
I would say everything optional, except what is necessary for Navitia.
We will not have data for all countries (like the INSEE code, French code).

Reply:
everyhting is optional now except the id